README.md

```diff
@@ -38,34 +38,20 @@ multipass help <command>
 
 Here's a set of steps to build and run your own build of multipass:
 
-## Build Dependencies
+## Install snapcraft
 
 ```
-cd <multipass>
-apt install devscripts equivs
-mk-build-deps -s sudo -i
+sudo snap install snapcraft
 ```
 
-## Building
+## Build and install the snap
 
 ```
 cd <multipass>
-git submodule update --init --recursive
-mkdir build
-cd build
-cmake ../
-make
+snapcraft
```

Comment:
Failed on Bionic due to wrong qt version. cleanbuild should do  - but then needs lxd instructions added.

Reply:
bionic (18.04) is kind of mess, just tried building on clean install and got error
`Failed to fetch stage packages: Error downloading packages for part 'libqt5core5a': The package 'libdouble-conversion1v5' was not found..`
I already know that libprocess-cpp-dev is not available on bionic.

IMHO solution is to add xenial (16.04) to sources.list

Reply:
Yeah, I definitely would not recommend building the snap directly on the host system.  I would definitely  recommend adding `SNAPCRAFT_BUILD_ENVIRONMENT=multipass` or `SNAPCRAFT_BUILD_ENVIRONMENT=lxd` here.

Reply:
I believe that's taken care of by adding `base:` to snapcraft.yaml (which I now did).